source/agora/consensus/data/UtxoSet.d

```diff
@@ -0,0 +1,292 @@
+/*******************************************************************************
```

Comment:
Message mentions "UtxoSet" (UTXOSet), and the "hot cache" which have been removed.